buildpacks/dotnet/src/layers/sdk.rs

```diff
@@ -39,7 +39,7 @@ type HandleResult = Result<
     libcnb::Error<DotnetBuildpackError>,
 >;
 
-const MAX_RETRIES: u8 = 1;
+const MAX_RETRIES: u8 = 4;
```

Comment:
Since this is an improvement to user-facing behaviour (improved reliability), should it be mentioned in the changelog? (If only so if the change is released before any other changes land there isn't an empty changelog for the release?)

Reply:
Yes good call https://github.com/heroku/buildpacks-dotnet/pull/160/commits/7cd3fe6e42d35dcfef8eb1ed09506bebcd764a6a